### PR TITLE
Minor test suite refactor

### PR DESCRIPTION
**Pull request by ddanderson**
Added copy_live_wiredtiger_home function, with two callers.

### CODE REVIEW

File: test/suite/helper.py [hunk 1]
```diff
@@ -96,6 +96,29 @@ def confirm_empty(self, uri):
...
+    # unaligned copy requires 'dd', which may not be available on Windows
+    if not aligned and os.name == "nt":
+        raise AssertionError(
+            'copy_live_wiredtiger_home: unaligned copy impossible on Windows')
+    shutil.rmtree(newdir, ignore_errors=True)
+    os.mkdir(newdir)
+    for fname in os.listdir(olddir):
+        fullname = os.path.join(olddir, fname)
+        # Skip lock file, on Windows it is locked.
+        if os.path.isfile(fullname) and "WiredTiger.lock" not in fullname:
```
Comment (sueloverso) [inline test/suite/helper.py]: One difference between the original and the new code is that the original did shutil.copytree on non-windows systems to simulate the expected `cp -rp` that a user would do.  The new code now copies files individually and skips the `WiredTiger.lock` file.  So we're no longer testing what a user is likely to do.  @michaelcahill is that difference important?

**Comment by sueloverso** [comment]
Note this was SERVER-18899 (I thought I had typed this comment already but it disappeared).  Please fix the commit line so it correctly appears in the ticket.

**Comment by ddanderson** [comment]
Yuck.  I already merged it. :-(  Not sure how to fix.

File: test/suite/helper.py (hunk 1 above)
Comment (michaelcahill) [inline test/suite/helper.py]: This is what a user would have to do on Windows, so I think it's fine.  It's also a reasonable implementation of `cp -rp`, so I don't think the difference will matter.

**Pull request merged**